user/configure-device.rst

```diff
@@ -147,20 +146,19 @@ Save the file and start openwisp-config:
 
     /etc/init.d/openwisp_config restart
 
-Your OpenWRT instance should register itself to your openwisp2 controller.
+Your OpenWRT instance should register itself to your OpenWISP2 controller.
```

Comment:
What if we stop calling it OpenWISP2 Controller and just call it OpenWISP Controller?